xbmc/interfaces/builtins/AddonBuiltins.cpp

```diff
@@ -43,6 +44,28 @@
 
 using namespace ADDON;
 
+/*! \brief Install an addon.
+ *  \param params The parameters.
+ *  \details params[0] = add-on id.
+ */
+static int InstallAddon(const std::vector<std::string>& params)
+{
+  if (params.size())
```

Comment:
Unnecessary. the number you enter in GetOperations ensures the parameter is there